app/components/list-item.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class ListItemComponent extends Component {
+    @tracked isCompleted = false;
```

Comment:
This violates the single responsibility pattern because this is a visual component that contains mutations to the model (description, due date, etc).

Keeping the 'isCompleted' status alongside the data such as the description, completed date, etc. will help in multiple ways.

For example, if you wanted to say the data to the localstorage it is going to be a lot easier.